From 84b2a35af14b7c4756e6b40db6237cfbf3208fd6 Mon Sep 17 00:00:00 2001
From: Sabri Ünal <yakushabb@gmail.com>
Date: Mon, 1 Jul 2024 13:00:45 +0300
Subject: [PATCH] appdata: Fix appdata issues

- Add missing developer block
- Remove .desktop extension from the ID
- Add vcs-browser and translate URLs
- Add provides and replaces tags to identify the old ID
---
 data/io.github.dyegoaurelio.simple-wireplumber-gui.appdata.xml.in | 13 ++++++++++++-
 1 file changed, 12 insertions(+), 1 deletion(-)

diff --git a/data/io.github.dyegoaurelio.simple-wireplumber-gui.appdata.xml.in b/data/io.github.dyegoaurelio.simple-wireplumber-gui.appdata.xml.in
index b936ac7..dcffa6d 100644
--- a/data/io.github.dyegoaurelio.simple-wireplumber-gui.appdata.xml.in
+++ b/data/io.github.dyegoaurelio.simple-wireplumber-gui.appdata.xml.in
@@ -1,8 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
 	<name>Simple Wireplumber GUI</name>
-	<id>io.github.dyegoaurelio.simple-wireplumber-gui.desktop</id>
+	<id>io.github.dyegoaurelio.simple-wireplumber-gui</id>
 	<developer_name>Dyego Aurélio</developer_name>
+	<developer id="io.github.dyegoaurelio">
+		<name>Dyego Aurélio</name>
+	</developer>
 	<screenshots>
 		<screenshot type="default">
 			<image>
@@ -30,6 +33,8 @@
 
 	<url type="homepage">https://github.com/dyegoaurelio/simple-wireplumber-gui</url>
 	<url type="bugtracker">https://github.com/dyegoaurelio/simple-wireplumber-gui/issues</url>
+	<url type="vcs-browser">https://github.com/dyegoaurelio/simple-wireplumber-gui</url>
+	<url type="translate">https://github.com/dyegoaurelio/simple-wireplumber-gui/tree/main/po</url>
 
 	<summary>
 		A simple GTK4 GUI for PipeWire
@@ -41,5 +46,11 @@
 	<content_rating type="oars-1.1" />
 
 	<launchable type="desktop-id">io.github.dyegoaurelio.simple-wireplumber-gui.desktop</launchable>
+	<provides>
+		<id>io.github.dyegoaurelio.simple-wireplumber-gui.desktop</id>
+	</provides>
+	<replaces>
+		<id>io.github.dyegoaurelio.simple-wireplumber-gui.desktop</id>
+	</replaces>
 
 </component>
--
libgit2 1.7.2

